### PR TITLE
[Android] Splash screen implementation

### PR DESCRIPTION
**Pull request by vikvelma**
**Implemented splash screen**
- Brand logo on the splash screen slides up to center.
- At end of animation it navigates to `LoginActivity`

[splash_phone.webm.zip](https://github.com/HelloImKevo/SkeletonPaymentApp-Senior/files/4394139/splash_phone.webm.zip)

### CODE REVIEW

File: Android/app/src/main/res/values/config.xml [hunk 1]
```diff
@@ -6,4 +6,5 @@
 
     <!-- Animation Values -->
     <item name="anim_duration" format="integer" type="integer">500</item>
+    <item name="slid_up_anim_duration" format="integer" type="integer">1000</item>
```
Comment (HelloImKevo) [inline Android/app/src/main/res/values/config.xml]: This looks like an accidental typo - can you change this to `slide`

**Review by HelloImKevo — CHANGES_REQUESTED** [review]

**Comment by HelloImKevo** [comment]
This looks great! Small request noted on the first commit - it took me a few minutes, but I figured out how to "patch" the changes to test it locally - from the origin repo, I did this:
```
# List all remote pull requests
git ls-remote origin 'pull/*/head'

# Fetch a specific pull request into a local branch and give it a custom name
git fetch origin pull/3/head:pr-splash-screen
```

Gonna add these ^ notes in the README

File: Android/app/src/main/res/values/config.xml [hunk 2]
```diff
@@ -6,5 +6,5 @@
 
     <!-- Animation Values -->
     <item name="anim_duration" format="integer" type="integer">500</item>
-    <item name="slid_up_anim_duration" format="integer" type="integer">1000</item>
+    <item name="slide_up_anim_duration" format="integer" type="integer">1000</item>
```
Comment (HelloImKevo) [inline Android/app/src/main/res/values/config.xml:9]: Hey Vik - I think the refactor was missed - doesn't this need to change in `Android/app/src/main/res/anim/slide_up.xml` also?

Comment (vikvelma) [reply Android/app/src/main/res/values/config.xml:9]: Sorry for the stupidity!! 😞

Comment (HelloImKevo) [reply Android/app/src/main/res/values/config.xml:9]: No problemo! 😁

**Review by HelloImKevo — APPROVED** [review]

**Pull request merged**